war/src/main/java/io/lumeer/engine/rest/CollectionService.java

```diff
@@ -144,17 +148,17 @@ public void init() {
     * @return Collection with provided code
     */
    @GET
-   @Path("/{collectionCode}/")
+   @Path("{collectionCode}")
    public Collection getCollection(final @PathParam("collectionCode") String collectionCode) throws CollectionNotFoundException, UnauthorizedAccessException {
       if (collectionCode == null) {
          throw new BadRequestException();
       }
 
-      if (!collectionFacade.hasCollection(collectionCode)) {
+      Collection collection = collectionFacade.getCollection(collectionCode);
+      if (collection == null) {
          throw new UserCollectionNotFoundException(ErrorMessageBuilder.userCollectionNotFoundString(collectionCode));
       }
-
-      if (!securityFacade.hasCollectionRole(projectCode, collectionCode, LumeerConst.Security.ROLE_READ)) {
+      if (!collection.getUserRoles().contains(LumeerConst.Security.ROLE_MANAGE)) {
```

Comment:
Why does user need to have `manage` role in order to be able to read collection?